### PR TITLE
Adds a few more names to the name generator.

### PR DESCRIPTION
**Pull request by debayande**
**- What I did**
_Added a few more names to the name generator_

**- How I did it**
_I added the names to `pkg/namesgenerator/names-generator.go`_

**- How to verify it**
_Please check `pkg/namesgenerator/names-generator.go`_

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -617,12 +719,18 @@ var (
...
+		"wu",
+
 		// Rosalyn Sussman Yalow - Rosalyn Sussman Yalow was an American medical physicist, and a co-winner of the 1977 Nobel Prize in Physiology or Medicine for development of the radioimmunoassay technique. https://en.wikipedia.org/wiki/Rosalyn_Sussman_Yalow
 		"yalow",
 
 		// Ada Yonath - an Israeli crystallographer, the first woman from the Middle East to win a Nobel prize in the sciences. https://en.wikipedia.org/wiki/Ada_Yonath
 		"yonath",
 
+		// Tu Youyou - Chinese pharmaceutical chemist and educator known for discovering artemisinin and dihydroartemisinin, used to treat malaria, which has saved millions of lives. Joint winner of the 2015 Nobel Prize in Physiology or Medicine. https://en.wikipedia.org/wiki/Tu_Youyou
+		"youyou",
```
Comment (AkihiroSuda) [inline pkg/namesgenerator/names-generator.go]: Her family name is Tu

Comment (debayande) [reply pkg/namesgenerator/names-generator.go]: Oops! Will fix right away.

**Comment by AkihiroSuda** [comment]
Please rebase?

**Comment by AkihiroSuda** [comment]
@thaJeztah PTAL?

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: pkg/namesgenerator/names-generator.go [hunk 2]
```diff
@@ -186,23 +192,41 @@ var (
 		// Emmett Brown invented time travel. https://en.wikipedia.org/wiki/Emmett_Brown (thanks Brian Goff)
 		"brown",
 
+		// Linda Brown Buck - American biologist and Nobel laureate best known for her genetic and molecular analyses of the mechanisms of smell. https://en.wikipedia.org/wiki/Linda_B._Buck
+		"buck",
+
+		// Susan Jocelyn Bell Burnell - Northern Irish astrophysicist who discovered radio pulsars and was the first to analyse them. https://en.wikipedia.org/wiki/Jocelyn_Bell_Burnell
+		"burnell",
```
[the remark this answers is not in the document]
Comment (debayande) [reply pkg/namesgenerator/names-generator.go]: That would conflict with `bell` from Alexander Graham Bell's surname.

Despite her divorce, Dame Jocelyn chooses to keep that surname. For example, she is the patron of the [Burnell House](https://en.wikipedia.org/wiki/Cambridge_House_Grammar_School#The_House_System) in the Cambridge House Grammar School. Also [a book](https://www.amazon.co.uk/Dark-Matter-Poems-Maurice-Riordan/dp/190308010X) co-edited by her as recently as in 2008 (she divorced Martin Burnell in 1993), retains _Burnell_ in her name. So, I think `burnell` should be fine.

[the remark this answers is not in the document]
Comment (debayande) [reply pkg/namesgenerator/names-generator.go]: That would conflict with _bell_ from Alexander Graham Bell's surname.

Despite her divorce, Dame Jocelyn chooses to keep that surname. For example, she is the patron of the [Burnell House](https://en.wikipedia.org/wiki/Cambridge_House_Grammar_School#The_House_System) in the Cambridge House Grammar School. Also [a book](https://www.amazon.co.uk/Dark-Matter-Poems-Maurice-Riordan/dp/190308010X) co-edited by her as recently as in 2008 (she divorced Martin Burnell in 1993), retains Burnell in her name. So, I think _burnell_ should be fine.

File: pkg/namesgenerator/names-generator.go [hunk 3]
```diff
@@ -186,23 +192,41 @@ var (
 		// Emmett Brown invented time travel. https://en.wikipedia.org/wiki/Emmett_Brown (thanks Brian Goff)
 		"brown",
 
+		// Linda Brown Buck - American biologist and Nobel laureate best known for her genetic and molecular analyses of the mechanisms of smell. https://en.wikipedia.org/wiki/Linda_B._Buck
+		"buck",
+
+		// Susan Jocelyn Bell Burnell - Northern Irish astrophysicist who discovered radio pulsars and was the first to analyse them. https://en.wikipedia.org/wiki/Jocelyn_Bell_Burnell
```
[the remark this answers is not in the document]
Comment (debayande) [reply pkg/namesgenerator/names-generator.go]: Done!

File: pkg/namesgenerator/names-generator.go [hunk 4]
```diff
@@ -186,23 +192,41 @@ var (
...
+		"burnell",
+
 		// Rachel Carson - American marine biologist and conservationist, her book Silent Spring and other writings are credited with advancing the global environmental movement. https://en.wikipedia.org/wiki/Rachel_Carson
 		"carson",
 
+		// Mary Lucy Cartwright - British mathematician who was one of the first to study what is now known as chaos theory. Also known for Cartwright's theorem which finds applications in signal processing. https://en.wikipedia.org/wiki/Mary_Cartwright
```
[the remark this answers is not in the document]
Comment (debayande) [reply pkg/namesgenerator/names-generator.go]: Done!

File: pkg/namesgenerator/names-generator.go [hunk 5]
```diff
@@ -153,12 +153,18 @@ var (
 		// Karl Friedrich Benz - a German automobile engineer. Inventor of the first practical motorcar. https://en.wikipedia.org/wiki/Karl_Benz
 		"benz",
 
+		// Timothy John Berners-Lee - English computer scientist and engineer, best known as the inventor of the World Wide Web. https://en.wikipedia.org/wiki/Tim_Berners-Lee
+		"berners",
```
[the remark this answers is not in the document]
Comment (debayande) [reply pkg/namesgenerator/names-generator.go]: Actually, we have: cf. [al-Battānī](https://github.com/debayande/moby/blob/fc09683d601f7374bf5ded4767603eb289ca7466/pkg/namesgenerator/names-generator.go#L108), [Joliot-Curie](https://github.com/debayande/moby/blob/fc09683d601f7374bf5ded4767603eb289ca7466/pkg/namesgenerator/names-generator.go#L422), [Payne-Gaposchkin](https://github.com/debayande/moby/blob/fc09683d601f7374bf5ded4767603eb289ca7466/pkg/namesgenerator/names-generator.go#L590), [Levi-Montalcini](https://github.com/debayande/moby/blob/fc09683d601f7374bf5ded4767603eb289ca7466/pkg/namesgenerator/names-generator.go#L620), [Barré-Sinoussi](https://github.com/debayande/moby/blob/fc09683d601f7374bf5ded4767603eb289ca7466/pkg/namesgenerator/names-generator.go#L662), [Nüsslein-Volhard](https://github.com/debayande/moby/blob/fc09683d601f7374bf5ded4767603eb289ca7466/pkg/namesgenerator/names-generator.go#L719).

In each of these cases, the committer has chosen to go either with the first or the second of the two dashed surnames. I've chosen to go with the first.

File: pkg/namesgenerator/names-generator.go [hunk 6]
```diff
@@ -473,6 +575,9 @@ var (
 		// Robert Noyce invented silicone integrated circuits and gave Silicon Valley its name. - https://en.wikipedia.org/wiki/Robert_Noyce
 		"noyce",
 
+		// Heike Kamerlingh Onnes - Dutch physicist and Nobel laureate; was the first to liquefy helium. Also discovered superconductivity. https://en.wikipedia.org/wiki/Heike_Kamerlingh_Onnes
+		"onnes",
```
[the remark this answers is not in the document]
Comment (debayande) [reply pkg/namesgenerator/names-generator.go]: Replaced by [Ralph C. Merkle](https://github.com/debayande/moby/blob/02a7afa7ac6a99527765b0e7ce77f4268f6b9e01/pkg/namesgenerator/names-generator.go#L536).

**Comment by debayande** [comment]
@thaJeztah I've made the changes you requested. Please take a look. Thanks!

File: pkg/namesgenerator/names-generator.go (hunk 5 above)
[the remark this answers is not in the document]
Comment (thaJeztah) [reply pkg/namesgenerator/names-generator.go]: Hm, looks like we're not really consistent there, or the committer may have misinterpreted double names as maiden names. For al-Battānī, I see that `al-` is more of a "prefix" (so possibly that's commonly excluded / not part of the last name). For Joliot-Curie,  Payne-Gaposchkin, Barré-Sinoussi, and Nüsslein-Volhard, I see their maiden-names were used, but "Levi-Montalcini" may be incorrect

[the remark this answers is not in the document]
Comment (debayande) [reply pkg/namesgenerator/names-generator.go]: Replaced by [Horst Feistel](https://en.wikipedia.org/wiki/Horst_Feistel).

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Comment by debayande** [comment]
☝️ Just squashed two commits into one; no new deltas have been introduced.

**Review by vdemeester — APPROVED** [review]
LGTM :tiger:

**Comment by thaJeztah** [comment]
Merged! Thanks for contributing @debayande !!

**Pull request merged**